digital-deck-server/socket.js

```diff
@@ -6,9 +6,17 @@ function handleSocket(io)
       
         socket.on('joinRoom', (sessionId) => {
             socket.join(sessionId);
-            console.log("in server", sessionId)
+            console.log("in server", sessionId);
+        });
+
+        socket.on('disconnect', () => {
+            console.log("Disconnected");
+        });
+
+        socket.on('drawCard', ({ sessionId, numCards, player }) => {
+            // console.log("current cards", cards);
+            socket.to(sessionId).emit('cardDrawn', { deck: numCards, player: player });
```

Comment:
Can you please confirm that integers will work as room names?

Reply:
It does. I'm pretty sure Hassan also confirmed it when working on his ticket